ecs-cli/modules/cli/compose/adapter/containerconfig.go

```diff
@@ -28,6 +28,7 @@ type ContainerConfig struct {
 	MountPoints           []*ecs.MountPoint
 	PortMappings          []*ecs.PortMapping
 	Privileged            bool
+	PseudoTerminal        bool
```

Comment:
In order for `pseudoTerminal` to work, it looks like [`interactive` needs to be set to true](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#container_definition_interactive). Do we want to add support for that field in this PR as well, or always set it depending on the `tty` value?

Reply:
I verified with the agent team that it's meant to be the other way around. Regardless it seems like both flags can be set independently.